src/dev/docker/Dockerfile

```diff
@@ -149,7 +149,7 @@ COPY kernels /usr/local/share/jupyter/kernels
 COPY kernels/ir/Rprofile.site /usr/local/sage/local/lib/R/etc/Rprofile.site
 
 # Build a UTF-8 locale, so that tmux works -- see https://unix.stackexchange.com/questions/277909/updated-my-arch-linux-server-and-now-i-get-tmux-need-utf-8-locale-lc-ctype-bu
-RUN apt -y install locales && echo "en_US.UTF-8 UTF-8" > /etc/locale.gen && locale-gen
+RUN echo "en_US.UTF-8 UTF-8" > /etc/locale.gen && locale-gen
```

Comment:
I think locales-all is already installed.

The issue is that locales have to be compiled (it's what locale-gen does) and that takes a lot of space, that's why it is done on a request basis.

Compiling `en_US.UTF-8` takes 1.68M, I'm not sure if other languages may take more space. There are almost 500 locales, so I guess at least 500-1000 M in total.

If you were to try, I would replace the above by something like
```
RUN sed -i 's/^#//' /etc/locale.gen && locale-gen
```
and see how much space that takes.

I think it's more useful to have it just like that and add a section to the documentation on how to compile more locales, I guess something like:
```
docker exec -it cocalc "echo 'es_UY.UTF-8 UTF-8'> /etc/locale.gen && locale-gen"
```
(UNTESTED)